src/containers/ToDoList/ToDoList.jsx

```diff
@@ -151,6 +166,39 @@ export const ToDoList = () => {
             <ThemeChangingIcon style={{ fontSize: 40, color: "#fff" }} />
           </button>
         </StyledWrapperTitle>
+
+        <StyledWrapperCompletedTasks>
+          <span>Completed tasks</span>
+
+          <Box sx={{ display: "flex", alignItems: "center" }}>
+            <Box sx={{ width: "100%", mr: 1 }}>
+              <LinearProgress
+                variant="determinate"
+                value={
+                  totalTasksCount > 0
+                    ? (numberOfCompletedTasks / totalTasksCount) * 100
+                    : 0
+                }
+                sx={{
+                  width: "100%",
+                  "& .MuiLinearProgress-bar": {
+                    backgroundColor: "#2e9acc",
+                  },
+                  backgroundColor: "#b4b4b8",
+                }}
```

Comment:
This should go to const `progressBarStyles` not to mess with the JSX part of the file. In other words - for better readability